pkgs/misc/mlxbf-bootimages/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, lib
+, fetchurl
+, dpkg
+}:
+
+let
+  mainVersion = "4.0.2";
```

Comment:
I would rather put mainVersion in the version and compute mainVersion from version so you can override properly.

Reply:
Good point.